app/main.py

```diff
@@ -2,5 +2,18 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_dict = {}
+
+    def wrapper(*args, **kwargs) -> Callable:
```

Comment:
The outcome of the wrapper is not always `Callable`, it could be `Any` and depends on the wrapped function logic. Change your return type annotation.